content/en/runtime.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, is something that executes a piece of software.
```

Comment:
```suggestion
A runtime, in general, executes a piece of software.
```

Reply:
I was considering this edit too, I think you're right, let's remove it.